src/main/java/com/wolox/training/models/Book.java

```diff
@@ -79,7 +82,7 @@ public String getImage() {
   }
 
   public void setImage(String image) {
-    this.image = image;
+    this.image = checkNotNull(image, Constant.NOT_NULL_MESSAGE, "image");
   }
```

Comment:
Add check so that the attributes image, genre, author, title, subtitle, publisher and year are not empty.
Example:
Preconditions.checkArgument(!genre.isEmpty(), "The genre field cannot be empty"); 

Remember to add this validation after checking if it is null.